src/Microsoft.Framework.Configuration/Helper/ConfigurationHelper.cs

```diff
@@ -10,19 +10,11 @@ public static class ConfigurationHelper
     {
         public static string ResolveConfigurationFilePath(IConfigurationBuilder configuration, string path)
         {
+            string basePath = configuration.Properties["BasePath"].ToString();
```

Comment:
Configuration "core" should become ignorant of any properties, including BasePath. Any common logic to get the base path or to resolve it should move to the new file-based package.